spec/miscellaneous/lib/tasks/after_tasks.rake_spec.rb

```diff
@@ -46,3 +46,67 @@
     end
   end
 end
+
+describe "rake After:unhide_invited_works" do
+  let(:anonymous_collection) { create (:anonymous_collection) }
+  let(:unrevealed_collection) { create (:unrevealed_collection) }
+  let(:other_collection) { create (:collection) }
+  let(:anonymous_unrevealed_collection) { create (:anonymous_unrevealed_collection)}
+  let(:anonymous_work) { create(:posted_work, collections: [anonymous_collection]) }
+  let(:unrevealed_work) { create(:posted_work, collections: [unrevealed_collection]) }
+  let(:normal_work) { create(:posted_work, collections: [other_collection]) }
+  let(:anonymous_unrevealed_work) { create(:posted_work, collections: [anonymous_unrevealed_collection]) }
```

Comment:
It would be a good idea to have some works that are actually supposed to be anonymous or unrevealed as well, just to make sure we don't accidentally de-anon or reveal them. So there could be `anonymous_work`, `anonymous_invited_work`, `unrevealed_work`, `unrevealed_invited_work`, etc.

Reply:
Sure :)